rclpy/rclpy/__init__.py

```diff
@@ -12,12 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import rclpy._rclpy__rmw_opensplice_cpp as _rclpy
+import os
 
+from rclpy import selector
 from rclpy.node import Node
 
+_rclpy = None
+
 
 def init(args):
+    global _rclpy
+    rclpy_rmw_env = os.getenv('RCLPY_IMPLEMENTATION')
+    if rclpy_rmw_env:
```

Comment:
`is not None`? Should there be a different message if it is an empty string, can it be an empty string?